src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -145,10 +145,10 @@ def unskip_table_or_view(
             table_name (str): The table name of the table to be unskipped.
             load_table (Callable[[str, str], Table | None]): A function that loads a table from the metastore.
         """
+        table = load_table(schema_name, table_name)
+        if table is None:
+            raise NotFound("[TABLE_OR_VIEW_NOT_FOUND]")
```

Comment:
This would be: 

```suggestion
           logger.error(
                f"Failed to remove skip marker from table: {schema_name}.{table_name}. Table not found.",
            )
```

If you want to avoid duplication of the message, create a: `table_not_found_message = f"Failed to remove skip marker from table: {schema_name}.{table_name}. Table not found."`